content/docs/user-guide/components/reference/physx/force-region.md

```diff
@@ -165,7 +165,7 @@ You can create a force region so that force applies to another entity that enter
 
```

Comment:
The *ForceRegion* entity described in the above steps is missing a dependency on the *PhysX Primitive Collider* component to be functional. I believe the *PhysX Static Rigid Body* component is expected here, so additional step with adding that component to the entity would clear that up.

Reply:
Well spotted! I'll add a step indicating to add a PhysX Static Rigid Body. Thanks.

Reply:
Thanks, looks good!